auth/options.go

```diff
@@ -32,7 +32,6 @@ type Option func(*Options)
 // Options contains options for configuring the behavior of the provider methods.
 // Not all providers/methods support all options.
 type Options struct {
-	Client          client.Client
```

Comment:
Client field removed to prevent ambiguity between explicit client parameter and Options.Client. 
All auth functions now require client.Client as an explicit mandatory parameter to ensure consistent usage and avoid unintended client selection.